annotations/views.py

```diff
@@ -883,4 +929,35 @@ def concept_details(request, conceptid):
     return HttpResponse(template.render(context))
 
 
+def sign_s3(request):
```

Comment:
This needs to be protected; use the [``@login_required`` decorator](https://docs.djangoproject.com/en/1.9/topics/auth/default/#django.contrib.auth.decorators.login_required).